src/Module/Admin/Themes/Embed.php

```diff
@@ -95,7 +95,7 @@ protected function content(array $request = []): string
 
 		$t = Renderer::getMarkupTemplate('admin/addons/embed.tpl');
 		return Renderer::replaceMacros($t, [
-			'$action' => DI::baseUrl()->get(true) . '/admin/themes/' . $theme . '/embed?mode=minimal',
+			'$action' => DI::baseUrl() . '/admin/themes/' . $theme . '/embed?mode=minimal',
```

Comment:
Since there can be no scheme difference, we can rely on `<base>` in all cases now.
```suggestion
			'$action' => 'admin/themes/' . $theme . '/embed?mode=minimal',
```